requirements.txt

```diff
@@ -4,4 +4,5 @@ scikit-learn>=0.19.1
 xlrd>=0.9.0
 pandas-flavor==0.1.2
 biopython
-requests
\ No newline at end of file
+requests
+unyt
```

Comment:
@rahosbach I just caught this one - we should keep `unyt` outside of `requirements.txt`, but it's ok for `unyt` to be in `requirements-dev.txt`. 

In a later PR, I will be removing `biopython` from `requirements.txt`, because it is technically an optional dependency.

Reply:
On the other hand, it should be added to `environment.yml`, because the Azure pipeline build uses all packages to run tests.

Reply:
OK.  I removed unyt from `requirements.txt` and added it to `environment-dev.yml`.  I didn't see an `environment.yml` file in the repo...